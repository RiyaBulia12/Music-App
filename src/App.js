import { useSelector } from 'react-redux';
import { Route, Routes } from 'react-router-dom';
import './assets/css/App.css';
import Navigation from './components/Navigation';
import Playlists from './components/Playlist';
import PlaylistSongs from './components/PlaylistSongs';

function App() {
  const playlistObj = useSelector((state) => state.playlist.playlistState);
  return (
    <>
      <Navigation />
      <Routes>
        <Route path="/" element={<Playlists playlist={playlistObj} />} />
        <Route path="/:playlistName/:playlistId" element={<PlaylistSongs />} />
      </Routes>
    </>
  );
}

export default App;
