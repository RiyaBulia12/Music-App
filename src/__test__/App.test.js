import { Provider } from 'react-redux';
import App from '../App';
import store from '../redux/configureStore';

test('App Test', () => {
  const MockApp = () => {
    <Provider store={store}>
      <App />
    </Provider>;
  };
  expect(MockApp).toMatchSnapshot();
});
