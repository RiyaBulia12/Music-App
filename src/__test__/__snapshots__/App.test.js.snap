// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Test 1`] = `[Function]`;
